app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -140,6 +140,33 @@ class OnboardingFragmentTest {
     }
   }
 
+  @Test
+  fun testOnboardingFragment_DefaultSlideTitleDescription_hasCorrectContentDescription() {
```

Comment:
follow the test naming convention and format - https://github.com/oppia/oppia-android/wiki/Oppia-Android-Testing#naming-convention

Reply:
As per the naming convention given, it's in the camelCase
`testAction_withOneCondition_withSecondCondition_hasExpectedOutcome `

`testOnboardingFragment_defaultSlide_correctContentDescription `

Reply:
Got it, just to maintain a uniformity with other tests, can we make it like ```testOnboardingFragment_defaultSlide_hasCorrectContentDescription``` ?

Reply:
sounds good

Reply:
Changes Done